docs/Other SDL Features/Remote Control Vehicle Features/index.md

```diff
@@ -26,52 +26,84 @@ Currently, the remote control feature supports these modules:
+| Seat-Back Vertical Position | backVerticalPosition | 0-100% | Get/Set/Notification | Adjust seat back height (in case there are two actuators for seat height), 0 means the lowest position, 100% means the highest position | Since SDL v5.0 |
+| Seat Back Tilt Angle | backTiltAngle | 0-100% | Get/Set/Notification | Backrest recline, 0 means the angle that back top is nearest to the steering wheel, 100% means the angle that back top is furthest from the steering wheel | Since SDL v5.0 |
+| Head Support Horizontal Positon | headSupportHorizontalPosition | 0-100% | Get/Set/Notification | Adjust head support forward/backward, 0 means the nearest position to the front, 100% means the furthest position from the front | Since SDL v5.0 |
+| Head Support Vertical Position | headSupportVerticalPosition | 0-100% | Get/Set/Notification | Adjust head support height (up or down), 0 means the lowest position, 100% means the highest position | Since SDL v5.0 |
+| Seat Massaging Enabled | massageEnabled | true, false | Get/Set/Notification | Indicates whether massage is enabled for a seat | Since SDL v5.0 |
+| Massage Mode | massageMode | MassageModeData struct | Get/Set/Notification | list of massage mode of each zone | Since SDL v5.0 |
+| Massage Cushion Firmness | massageCushionFirmness | MassageCushionFirmness struct | Get/Set/Notification | list of firmness of each massage cushion | Since SDL v5.0 |
+| Seat memory | memory | SeatMemoryAction struct | Get/Set/Notification | seat memory | Since SDL v5.0 |
+
+#### Audio
+
+|Control Item | RPC Item Name | Value Range | Type | Comments | Version Changes |
+| --------------- | ------------ | ------------ | ------------ | ------------ | ------------ |
+| Audio Volume | volume | 0%-100% | Get/Set/Notification | The audio source volume level | Since SDL v5.0 |
+| Audio Source | source | PrimaryAudioSource enum | Get/Set/Notification | Defines one of the available audio sources | Since SDL v5.0 |
+| Keep Context | keepContext | true, false | Set only | control whether HMI shall keep current application context or switch to default media UI/APP associated with the audio source | Since SDL v5.0 |
+| Equilizer Settings | equalizerSettings | EqualizerSettings struct | Get/Set/Notification | Defines the list of supported channels (band) and their current/desired settings on HMI | Since SDL v5.0 |
+
+#### Light
+
+| Control Item | RPC Item Name | Value Range | Type | Comments | Version Changes |
+| --------------- | ------------ | ------------ | ------------ | ------------ | ------------ |
+| Light State | lightState | Array of LightState struct | Get/Set/Notification | | Since SDL v5.0 |
+
+#### HMI Settings
+
+| Control Item | RPC Item Name | Value Range | Type | Comments | Version Changes |
+| --------------- | ------------ | ------------ | ------------ | ------------ | ------------ |
+| Display Mode | displayMode | DAY, NIGHT, AUTO | Get/Set/Notification | Current display mode of the HMI display | Since SDL v5.0 |
```

Comment:
The value ranges in this table are all in all-caps. But the other tables just lowercase for the enum values (e.g. the *Radio State* has `Acquiring, acquired, multicast, not_found`)

Reply:
That was pre-existing, but it should be fixed on the latest commit